_datafiles/world/default/rooms/frostfang_slums/zone-config.yaml

```diff
@@ -0,0 +1,9 @@
+name: Frostfang Slums
+roomid: 439
+autoscale:
+  minimum: 5
+  maximum: 10
+mutators:
+- mutatorid: pvp-enabled
```

Comment:
Should this be indented?

Reply:
I think it should, but I also think our yaml library isn't consistent about formatting vs. the specification. I'll try and update these.

Reply:
Indented them.